### PR TITLE
[Pages] Functions style guide revision

### PR DESCRIPTION
**Pull request by deadlypants1973**

### CODE REVIEW

File: content/pages/platform/functions/advanced-mode.md [hunk 1]
```diff
@@ -1,14 +1,16 @@
...
+In some cases, Pages Functions' built-in file path based routing and middleware system is not desirable for existing applications. You may have a Worker that is complex and difficult to splice up into Pages' file-based routing system. For these cases, Pages offers the ability to define a `_worker.js` file in the output directory of your Pages project.
+
+When using a `_worker.js` file, the entire `/functions` directory is ignored, including its routing and middleware characteristics. Instead, the `_worker.js` file is deployed as is and must be written using the [Module Worker syntax](/workers/runtime-apis/fetch-event/#syntax-module-worker). If you have never used Module syntax, refer to the [JavaScript modules blog post](https://blog.cloudflare.com/workers-javascript-modules/) to learn more. Using Module Workers enables JavaScript frameworks to generate a Worker as part of the Pages output directory contents.
 
 Your custom Module Worker will assume full control of all incoming HTTP requests to your domain. Because of this, your custom Worker is required to make and/or forward requests to your project's static assets.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/advanced-mode.md]: This needs an H2. Is `## Set up a Module Worker` correct here? @WalshyDev
Are we telling people to make a new Worker project for advanced mode? Isn't advanced mode specifically for people who already have existing Workers? (broader question: I get the HTTP requests routing functionality, but why is this code block here?)

File: content/pages/platform/functions/advanced-mode.md [hunk 2]
```diff
@@ -35,7 +37,7 @@ export default {
 {{</tab>}}
 {{<tab label="ts">}}
 ```ts
-// Note: You would need to compile your TS into JS and output it as a _worker.js file. We don't read _worker.ts
+// Note: You would need to compile your TS into JS and output it as a `_worker.js` file. We do not read `_worker.ts`
```
Comment (deadlypants1973) [inline content/pages/platform/functions/advanced-mode.md:44]: who is we @WalshyDev

File: content/pages/platform/functions/advanced-mode.md [hunk 3]
```diff
@@ -58,12 +60,15 @@ export default {
...
 
-The `env.ASSETS.fetch()` function will allow you to send the user to a modified path which is defined through the `url` parameter. `env` is the object that contains your environment variables and bindings. `ASSETS` is a default Function binding that allows communication between your Function and Pages' asset serving resource. `fetch()` calls to Pages' asset-serving resource and serves the requested asset.
+* The `env.ASSETS.fetch()` function will allow you to send the user to a modified path which is defined through the `url` parameter. 
+* `env` is the object that contains your environment variables and bindings. 
+* `ASSETS` is a default Function binding that allows communication between your Function and Pages' asset serving resource. 
+* `fetch()` calls to Pages' asset-serving resource and serves the requested asset.
 
 {{<Aside type="warning">}}
 
 Your custom Module Worker is required to forward requests to static assets. Failure to do so will result in broken and/or unwanted behavior because your website's contents will not be served if you do not serve it.
```
Comment (deadlypants1973) [inline content/pages/platform/functions/advanced-mode.md]: Is this saying that: if you have a Pages project and want to develop with a `_workers.js` file but do not set up this routing traffic Worker above, things will not work? @WalshyDev

File: content/pages/platform/functions/_index.md [hunk 4]
```diff
@@ -7,10 +7,13 @@ layout: single
 # Functions 
 
 ## Overview
-With Pages, you can now build full-stack applications by executing code on the Cloudflare network with help from [Cloudflare Workers](https://workers.cloudflare.com/). Using Functions enables you to run server-side code to enable dynamic functionality without running a dedicated server. With Functions, you can introduce application aspects such as authenticating, handling form submissions, or working with middleware.
+
+Pages Functions allows you to build full-stack applications by executing code on the Cloudflare network with [Cloudflare Workers](https://workers.cloudflare.com/). With Functions, you can introduce application aspects such as authenticating, handling form submissions, or working with middleware. Use Functions to deploy server-side code to enable dynamic functionality without running a dedicated server.
```
Comment (kodster28) [inline content/pages/platform/functions/_index.md]:
```suggestion
Pages Functions allows you to build full-stack applications by executing code on the Cloudflare network with [Cloudflare Workers](/workers/). With Functions, you can introduce application aspects such as authenticating, handling form submissions, or working with middleware. Use Functions to deploy server-side code to enable dynamic functionality without running a dedicated server.
```

File: content/pages/platform/functions/advanced-mode.md (hunk 2 above)
Comment (kodster28) [reply content/pages/platform/functions/advanced-mode.md:44]: Existential!

**Review by kodster28 — APPROVED** [review]

**Pull request closed without merging**